README.md

```diff
@@ -117,21 +117,6 @@ logdna register <email>
 # now paste the Ingestion Key into the kubectl commands above
 ```
 
-## Kubernetes Logging
-
-Set up Kubernetes logging with with the LogDNA Agent using just 2 `kubectl` commands! We extract pertinent Kubernetes metadata including the
-pod name, container name, container id, namespace, and labels:
-
-```
-kubectl create secret generic logdna-agent-key --from-literal=logdna-agent-key=<YOUR LOGDNA INGESTION KEY>
-kubectl create -f https://raw.githubusercontent.com/logdna/logdna-agent/master/logdna-agent-ds.yaml
-```
-
-This automatically installs a logdna-agent pod into each node in your cluster and ships stdout/stderr from all containers, both application
-logs and node logs. Note: by default, the agent pod will collect logs from all namespaces on each node, including `kube-system`. View your logs
-at https://app.logdna.com. See [YAML file](https://raw.githubusercontent.com/logdna/logdna-agent/master/logdna-agent-ds.yaml) for additional options
-such as `LOGDNA_TAGS`.
-
 ### Upgrading to LogDNA Agent 2.0 for Kubernetes
```

Comment:
if we are removing that block, why are we keeping this one? I'm wondering...

Reply:
Well, this is for v2 which is still fully supported.  Man, this is confusing.

Reply:
Not a bad idea, @mdeltito , but we can't just link people to the repo, right?  They'd be interested in the docker image, not the code base, amirite?

Reply:
well I mean a link to the readme over there, which seems to have all the same info for the rust kube agent (installing on k8s via yaml files which specify the image)

Reply:
I agree to have `./docs` folder - it's time to have that - they may wanna see the code - so, providing links to the images and codebase and this explanation should be enough - thanks, @mdeltito, for great idea!

Reply:
done